packages/hooks/src/useDelayedTotalAccountBalance.ts

```diff
@@ -0,0 +1,31 @@
+import useRealTotalAssetCurrency from './useTotalAssetCurrency';
+import useExchangeRate from './useExchangeRate';
+import useThrottle from './useThrottle';
+
+type TUseDelayedTotalAccountBalance = {
+    balance?: number;
+    currency?: string;
+    account_type?: string;
+};
+
+const useDelayedTotalAccountBalance = (accounts: TUseDelayedTotalAccountBalance[], delay: number) => {
```

Comment:
That'd be good to leave a small documentation about this hook like the same way in other hooks (some of them have it, some of them don't 😞 )

Reply:
will add it. will add test for the new introduced hooks as well.